docs/tutorial/t-setup-environment.md

```diff
@@ -1,20 +1,17 @@
 ## Setup Kubernetes and Spark Client Snap
 
-We are working with Kubernetes distribution for Spark. So, to be able to work with Kubernetes, 
-we first need to install a kubernetes cluster. In this tutorial we will [MicroK8s](https://microk8s.io/), 
-which is the simplest production-grade conformant K8s. Lightweight and focused. 
-Single command install on Linux, Windows and macOS.
+In order to be able to work with Kubernetes,  first we first to set up a Kubernetes cluster. In this tutorial we use [MicroK8s](https://microk8s.io/), 
+which is the simplest production-grade conformant K8s.
 
 ### Setup MicroK8s
 
-To install MicroK8s
+MicroK8s snap installation goes as simple as
 
 ```bash
 sudo snap install microk8s --classic
 ```
 
-It is also recommended to configure MicroK8s, so that the user has admin permission on 
-the cluster. Use the following commands to configure the user to perform actions on the K8s cluster:
+It is also recommended to configure MicroK8s in a way, so that the correpsonding user has admin rights on  the cluster. 
```

Comment:
spelling - "corresponding"

Reply:
also, this needs further explanation - why does the corresponding user need admin rights?